nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/api/validation/TestDocumentValidationActivation.java

```diff
@@ -107,6 +112,16 @@ public void testOnCreateDocumentActivationWithViolationNoForcing() {
         doc = session.createDocument(doc);
     }
 
+    // NXP-23256
```

Comment:
Good intention but such a comment is easily lost by merges. Move it inside the method or use Javadoc comments.